ElementX/Sources/FlowCoordinators/AuthenticationFlowCoordinator.swift

```diff
@@ -103,6 +102,21 @@ class AuthenticationFlowCoordinator: FlowCoordinatorProtocol {
         navigationRootCoordinator.setRootCoordinator(navigationStackCoordinator)
     }
     
+    private func startQrCodeLogin() {
```

Comment:
Only for consistency.
```suggestion
    private func startQRCodeLogin() {
```